cssparse/shared/src/test/scala/cssparse/PrettyPrinterTests.scala

```diff
@@ -0,0 +1,332 @@
+package cssparse
+
+import utest._
+
+object PrettyPrinterTests extends TestSuite {
+  val tests = this {
+    'basic {
+      'test1 {
+        val input =
```

Comment:
Could you add another test input, which is basically the same thing as `input` but fed into https://cssminifier.com/, and have an assert to make sure that both outputs parse and match the `output` expected